Source/Core/Core/HW/VideoInterface.h

```diff
@@ -447,6 +451,9 @@ class VideoInterfaceManager
   u32 m_even_field_last_hl = 0;   // index last halfline of the even field
   u32 m_odd_field_last_hl = 0;    // index last halfline of the odd field
 
+  float m_config_vi_freq_factor = 0.0f;
```

Comment:
```suggestion
  float m_config_vi_freq_factor = 1.0f;
```

Reply:
I'm simply doing the same as the CPU Overclock:

https://github.com/dolphin-emu/dolphin/blob/8ee64a84c7365b042f50e206d245b8a8464d1656/Source/Core/Core/CoreTiming.h#L201